WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -447,6 +453,37 @@ public void run() {
         }
     }
 
+    private void loadLocalVideoThumbnail(final String filePath, final WPNetworkImageView imageView) {
```

Comment:
Would you mind to investigate if `ImageUtils.getVideoFrameFromVideo` can be used in this case?
We're using it in Aztec and it seems to work very fine for both remote and local videos.